core/backend/src/test/categories/Category.test.ts

```diff
@@ -20,6 +20,7 @@ describe("Category", () => {
       projectExtents: { low: { x: -500, y: -500, z: -50 }, high: { x: 500, y: 500, z: 50 } },
       guid: Guid.createValue(),
     });
+    imodel.channels.addAllowedChannel("shared");
```

Comment:
I think you should add the shared channel to `StandaloneDb.createEmpty`. That function creates a new empty file that can only be edited in one session. There's no problem with editing the shared channel in that case. Then none of these tests will need to be changed.